<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGeneralSetting</class>
 <widget class="QDialog" name="TestGeneralSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>常规设置</string>
  </property>
  <widget class="QLabel" name="sdkVersionLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SDK Version：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="userControlButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>600</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>客户控制</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="clearButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>优先清晰</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>431</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="speedTestLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="logDirPathLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="setLogCompressEnabled">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>压缩Log </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableSmallVideoStream">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>670</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开启双路编码</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setLogDirPathButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置Log保存路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="sdkVersionLabel_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置 Log 输出级别：</string>
   </property>
  </widget>
  <widget class="QWidget" name="localView" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cloudControlButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>云控流控</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>431</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sdkVersionLabel_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>画质偏好</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="setLogLevelComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>输出所有级别的 Log</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>输出 DEBUG等</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>输出 INFO等</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>只输出WARNING，ERROR 和 FATAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>只输出ERROR 和 FATAL </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>只输出 FATAL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>不输出任何 SDK Log</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="smallButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>670</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>默认观看低清</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="startSpeedTest">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>网络测速</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="startCamTestButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>摄像头测试：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="setConsoleEnabled">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>控制台打印日志</string>
   </property>
  </widget>
  <widget class="QLabel" name="sdkVersionLabel_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>流控方案</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="smoothButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>优先流畅</string>
   </property>
  </widget>
  <zorder>listView_3</zorder>
  <zorder>listView</zorder>
  <zorder>listView_2</zorder>
  <zorder>listView_4</zorder>
  <zorder>listView_5</zorder>
  <zorder>sdkVersionLabel</zorder>
  <zorder>userControlButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>logDirPathLineEdit</zorder>
  <zorder>setLogCompressEnabled</zorder>
  <zorder>enableSmallVideoStream</zorder>
  <zorder>setLogDirPathButton</zorder>
  <zorder>sdkVersionLabel_4</zorder>
  <zorder>cloudControlButton</zorder>
  <zorder>sdkVersionLabel_5</zorder>
  <zorder>setLogLevelComboBox</zorder>
  <zorder>smallButton</zorder>
  <zorder>startSpeedTest</zorder>
  <zorder>startCamTestButton</zorder>
  <zorder>setConsoleEnabled</zorder>
  <zorder>sdkVersionLabel_6</zorder>
  <zorder>smoothButton</zorder>
  <zorder>speedTestLabel</zorder>
  <zorder>localView</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
